charts/istio-slo-validation/values.yaml

```diff
@@ -0,0 +1,70 @@
+# Prometheus URL
```

Comment:
I am surprised to see `Endpoint` and `istio/error-rate` here. I would have expected these to be empty and expect the user to fill in. Endpoint probably as required.

---

Would lower case `endpoint` be better?